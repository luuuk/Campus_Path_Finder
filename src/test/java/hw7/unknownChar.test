# This test checks the Graph's response to an unknown character

LoadGraph test test.tsv

FindPath test a k

FindPath test k a

FindPath test k k

