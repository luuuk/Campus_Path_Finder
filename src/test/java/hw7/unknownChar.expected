# This test checks the Graph's response to an unknown character

loaded graph test

unknown character k

unknown character k

unknown character k
unknown character k

